### PR TITLE
Demographics Mixin for PSRT Reports  + Render names dynamically

### PR DESCRIPTION
**Pull request by geek911**

### CODE REVIEW

File: esr21_reports/views/psrt_mixins/demographics_mixin.py [hunk 1]
```diff
@@ -0,0 +1,362 @@
...
+    @property
+    def site_ids(self):
+        site_ids = Site.objects.order_by('id').values_list('id', flat=True)
+        return site_ids
+
+
+    @property
+    def enrolled_pids(self):
+        enrolled = VaccinationDetails.objects.filter(
+            received_dose_before='first_dose').values_list(
```
Comment (ckgathi) [inline esr21_reports/views/psrt_mixins/demographics_mixin.py:29]: enrolled = VaccinationDetails.objects.filter(Q(received_dose_before='first_dose') | Q(received_dose_before='second_dose')).values_list('subject_identifier', flat=True)
enrolled = list(set(enrolled))

File: esr21_reports/views/graphs_mixins/enrollement_graph_mixin.py [hunk 2]
```diff
@@ -97,11 +98,18 @@ def get_context_data(self, **kwargs):
 		gender_by_site = self.get_vaccinated_by_site()
 		overall_participant = self.get_overall_participant()
 		overall_percentages = self.get_overall_percentage()
+		# site_names = self.sites_names
+
+		# import pdb
+		# pdb.set_trace()
```
Comment (ckgathi) [inline esr21_reports/views/graphs_mixins/enrollement_graph_mixin.py:104]: remove the comments for pdb

File: esr21_reports/views/psrt_mixins/demographics_mixin.py [hunk 3]
```diff
@@ -0,0 +1,362 @@
...
+        never_smoked = []
+        occasional_smoker = []
+        current_smoking = []
+        previous_smoker = []
+
+
+        for site_id in self.site_ids:
+
+            never_smoked_count = MedicalHistory.objects.filter(site_id=site_id, subject_visit__subject_identifier__in=self.enrolled_pids,
+                                                        smoking_status='never_smoked').values_list('subject_visit__subject_identifier').distinct().count()
```
Comment (ckgathi) [inline esr21_reports/views/psrt_mixins/demographics_mixin.py:195]: fix you pep8, lines too long
install pep8 or flake8
run command pep8

**Review by ckgathi — CHANGES_REQUESTED** [review]

File: esr21_reports/views/psrt_mixins/demographics_mixin.py (hunk 3 above)
Comment (geek911) [reply esr21_reports/views/psrt_mixins/demographics_mixin.py]: formatted the whole file

**Pull request merged**